viz/charts/linkerd-viz/templates/metrics-api.yaml

```diff
@@ -43,6 +43,7 @@ metadata:
   name: metrics-api
   namespace: {{ .Release.Namespace }}
 spec:
+  revisionHistoryLimit: {{.Values.revisionHistoryLimit}}
   replicas: {{.Values.metricsAPI.replicas}}
```

Comment:
It seems you're using a different ordering as what you did for the control-plane workloads. I know we might not be very consistent in this, but for new things it might be better to follow alphabetical ordering, so in this case placing `revisionHistoryLimit` after `replicas`.

Reply:
done